### PR TITLE
WebPreview: prevent scrolling when web preview is open.

### LINKED ISSUE #316: Editor: Double scrollbars when preview window open
If I scroll the preview window, the document behind will also scroll. There needs to be a:

`html, body { height: 100%; overflow-y: hidden; }`

set when the preview (all modals actually) are open.

raised by apeatling

### PR DESCRIPTION
**Pull request by mtias**
Also removes extra scrollbars from body. See #3901.

Fixes #316

### CODE REVIEW

File: client/components/web-preview/index.jsx [hunk 1]
```diff
@@ -70,6 +71,10 @@ const WebPreview = React.createClass( {
 		if ( this.props.previewUrl !== 'about:blank' ) {
 			this.setIframeUrl( this.props.previewUrl );
 		}
+
+		if ( this.props.showPreview ) {
+			componentClasses( document.documentElement ).add( 'no-scroll' );
```
Comment (aduth) [inline client/components/web-preview/index.jsx]: Can we just start using [`classList`](http://caniuse.com/#feat=classlist) instead?

```js
document.documentElement.classList.add( 'no-scroll' );
```

**Comment by aduth** [comment]
Works as expected.

Would like to see my recommended change made, but otherwise looks good :+1:

File: client/components/web-preview/index.jsx (hunk 1 above)
Comment (mtias) [inline client/components/web-preview/index.jsx]: Good call.

**Comment by aduth** [comment]
:+1:

**Pull request merged**